opentracing/span.py

```diff
@@ -30,6 +30,31 @@ class SpanContext(object):
 
     EMPTY_BAGGAGE = {}  # TODO would be nice to make this immutable
 
+    def to_trace_id(self):
+        """
+        Return the ID of the trace.
+
+        Globally unique. Every :class:`Span` in a trace shares this ID.
+        The empty string is a valid return value, while ``None`` is not.
+
+        :rtype: str
+        :return: the trace ID for this context.
+        """
+        return ''
+
+    def to_span_id(self):
+        """
+        Return the ID of the associated :class:`Span`.
+
+        Unique within a trace. Each :class:`Span` within
+        a trace contains a different ID.
+        The empty string is a valid return value, while ``None`` is not.
```

Comment:
Don't recall if the spec imposes any restrictions on the allowed alphabet of the output, like "must be URL safe"?

Reply:
Oh, that's a good point. I think the spec doesn't mention alphabet nor URL safety anywhere. I guess we should mention this bit there, at least ;)

Reply:
I don't think we can require url safety in the spec, so ok